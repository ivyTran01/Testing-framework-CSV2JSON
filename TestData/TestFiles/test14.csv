Ivy	22	true	null
