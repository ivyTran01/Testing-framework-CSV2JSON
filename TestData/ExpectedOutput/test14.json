[
 {
  "field1":"Ivy",
  "field2":22,
  "field3":true,
  "field4":null
 }
]